scripts/find-mem-leak.cocci

```diff
@@ -2,8 +2,8 @@
 expression x, y;
 statement S;
 @@
-  x = mbedtls_malloc(...);
-  y = mbedtls_malloc(...);
+  x = mbedtls_calloc(...);
+  y = mbedtls_calloc(...);
   ...
```

Comment:
@andresag01 This question is not related to the switch from `calloc` to `malloc`, but how does `find-mem-leak.cocci` help detecting memory leaks?